ios/browser/providers/BUILD.gn

```diff
@@ -35,6 +35,7 @@ group("brave_providers") {
     "//ios/chrome/browser/providers/text_zoom:chromium_text_zoom",
     "//ios/chrome/browser/providers/ui_utils:chromium_ui_utils",
     "//ios/chrome/browser/providers/url_rewriters:chromium_url_rewriters",
+    "//ios/chrome/browser/providers/user_feedback:chromium_user_feedback",
```

Comment:
do we need this actual dep or can we provide a dummy provider for this? cc @kylehickinson

Reply:
AFAIK we don't provide dummy providers for any of them, this file was only around to get rid of the MaterialComponents `+bundle` dep?

Reply:
not sure, but wondering if we can reduce the size of the output by replacing some of these. I guess follow up either way